### ISSUE TITLE
Remove requirement to tailor internal review template text

### ISSUE THREAD

**Issue by confirmordeny** [mysociety/whatdotheyknow-theme#1691]
The standard internal review text prompts the user to enter a complaint. There is no option to send something quick without the need for tailoring.

In most cases this would do:

"Please pass this on to the person who conducts Freedom of Information reviews. I am writing to request an internal review of [title of request]. **Please look into how my request was handled, whether all legal requirements were satisfied and whether any additional information ought to have been provided to me.**

A full history of my FOI request and all correspondence is available on the Internet at this address: [url]

Yours faithfully,"

**Comment by confirmordeny** [comment]
We could also change the workflow to say: "This text is based on our standard template for internal reviews and works well in most cases. You can add more specific arguments if you wish and this could increase your chances of getting more information released."

**Comment by mdeuk** [comment]
> "Please pass this on to the person who conducts Freedom of Information reviews. I am writing to request an internal review of [title of request]. **Please look into how my request was handled, whether all legal requirements were satisfied and whether any additional information ought to have been provided to me.**
> 

This _might_ not work in Scotland, where it is necessary to say [_why_ you wish a review](https://www.itspublicknowledge.info/asking-for-a-review).

_**Why?**_ A recent support case suggested that OSIC were unable to consider a complaint from a user, as they hadn't given an authority the reason for their review.

I don't think it is a blocker though - it improves on the existing template, and in the majority of cases, this ought to be enough to go with.


> We could also change the workflow to say: "This text is based on our standard template for internal reviews and works well in most cases. You can add more specific arguments if you wish and this could increase your chances of getting more information released."

+1

**Comment by HelenWDTK** [comment]
This issue is being closed due to a lack of discussion or resolution for over 12 months. Should we decide to revisit this issue in the future, it can be reopened.
